### PR TITLE
docs: update docs

### PR DESCRIPTION
**Pull request by radeva**
- update readme with more info about the forums and slack
- add PR template
- add CONTRIBUTING.md and DevelopmentWorkflow.md

### CODE REVIEW

File: DevelopmentWorkflow.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+1. Navigate to `demo/e2e`
+    ``` bash
+    cd demo/e2e
+    ```
+
+2. Make sure to have an emulator set up or connect a physical Android/iOS device.
+
+3. Build the app for Android or iOS
+    ```bash
+    tns run android/ios
```
Comment (lini) [inline DevelopmentWorkflow.md]: if only building, then command could be changed to 
```bash
tns build android
# or
ths build ios
```
Someone might try running `tns run android/ios` and get `The input is not valid sub-command for 'run' command`.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+contribution! 
+To help the rest of the community review your change, please follow the instructions in the template.
+-->
+
+<!-- PULL REQUEST TEMPLATE -->
+<!-- (Update "[ ]" to "[x]" to check a box) -->
+
+## PR Checklist
+
+- [ ] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
```
Comment (DimitarTachev) [inline .github/pull_request_template.md]: the PR title or the commit messages?

File: DevelopmentWorkflow.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+- [Run UI Tests](#run-ui-tests)
+- [Developer workflow](#developer-workflow)
+
+<!-- /TOC -->
+
+## Running locally
+
+### Prerequisites
+
+* Install your native toolchain and NativeScript as [described in the docs](https://docs.nativescript.org/plugins/plugins)
```
Comment (DimitarTachev) [inline DevelopmentWorkflow.md]: is this link correct?

File: DevelopmentWorkflow.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+* Review [NativeScript plugins documentation](https://docs.nativescript.org/plugins/plugins) for more details on plugins development
+
+
+### Install dependencies
+
+```
+$ cd nativescript-imagepicker/src
```
Comment (DimitarTachev) [inline DevelopmentWorkflow.md]: `npm run demo.android` will make `npm i` for you

File: DevelopmentWorkflow.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+```
+
+### Run the demo app
+From the `src` folder
+```
+$ npm run demo.ios 
+$ npm run demo.android
+```
+
+### Run the demo-angular app
```
Comment (DimitarTachev) [inline DevelopmentWorkflow.md]: why running the Angular app should be different from running the TS app?

File: README.md [hunk 6]
```diff
@@ -120,6 +139,11 @@ Once image is picked some options can be applied to it  before it is used:
...
+## Contribute
+We love PRs! Check out the [contributing guidelines](CONTRIBUTING.md). If you want to contribute, but you are not sure where to start - look for [issues labeled `help wanted`](https://github.com/NativeScript/nativescript-imagepicker/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22).
 
-2015, Telerik AD
\ No newline at end of file
+  
+## Get Help 
+Please, use [github issues](https://github.com/NativeScript/nativescript-imagepicker/issues) strictly for [reporting bugs](CONTRIBUTING.md#reporting-bugs) or [requesting features](CONTRIBUTING.md#requesting-new-features). For general questions and support, check out the [NativeScript community forum](https://discourse.nativescript.org/) or ask our experts in [NativeScript community Slack channel](http://developer.telerik.com/wp-login.php?action=slack-invitation).
+  
+![](https://ga-beacon.appspot.com/UA-111455-24/nativescript/nativescript-imagepicker?pixel)
```
Comment (DimitarTachev) [inline README.md]: what's this?

**Pull request merged**